Design : title animation 구현

ubninin opened the pull request:
# 📌개요

- 타이틀 애니메이션 추가, TitleManager 스크립트 수정
++ 추가 사항
- 애니메이션 수정, 코딩 컨벤션에 맞게 TitleManager 코드 스타일 개선

# 📜작업 내용

  - 화면 크기 1920x1080으로 수정
  - 로딩 진행 바, 퍼센트 텍스트 비활성화 (삭제 예정), LOADING... 텍스트로 대체함
  - 로딩 끝난 후 키 입력 받기 전 Title_BlinkPressAnyKeyText 애니메이션 재생
  - 로딩 끝난 후 아무 키나 입력 받으면 Title_Bounce 애니메이션 재생 후 다음 씬으로 넘어감
  - 전체 스크립트 파일 인코딩 형식 UTF-8로 변경, 한글 깨짐 해결
  - 2d sprite 패키지 설치
  - 
++ 추가 사항
  - 애니메이션 움직임, 키프레임, 애니메이터 컨트롤러 구조 일부 수정
  - 코드 내 변수 및 함수명을 명명 규칙에 따라 수정
  
# 📷관련 사진

## 수정 전
![image](https://github.com/user-attachments/assets/3c381077-537e-4712-bfdf-b3fc3c2b4e4b)
![image](https://github.com/user-attachments/assets/03df7eb8-f414-4c2a-9499-554236874748)
![image](https://github.com/user-attachments/assets/53ce3f49-ca49-4908-b234-65da050d9ed1)
![image](https://github.com/user-attachments/assets/bae4b0e3-48ab-4cc9-acf0-3242fd73abec)
로딩 완료 후 키 입력 대기, 키 입력 후 타이틀 전체 압축되었다가 발사

## ++ 수정 후
![image](https://github.com/user-attachments/assets/69a3ca5d-eb0c-434d-aefd-cd92abc8c281)
![image](https://github.com/user-attachments/assets/a1232988-4ba0-47bf-adc4-de4c9bba10a3)
압축된 후 로딩, 로딩 완료 후 키 입력 대기, 키 입력 후 로고 이미지만 발사되도록 수정


# 🧐문제

- 프로젝트를 처음 열었을 때 가끔씩 씬 뷰에는 UI가 보이지만 게임 뷰에서는 UI가 보이지 않는 문제가 있음 (다른 씬을 열었다가 다시 열면 정상적으로 보임)
- 로딩 진행 바, 퍼센트 텍스트를 혹시 몰라서 남겨두긴 했는데 완전히 삭제하는 게 좋을지 의견 부탁드립니다..

# 📜수정 사항 요약

- 애니메이션 움직임 수정
- TitleManager 코드 스타일 변경 (기능은 동일)

ubninin committed:
Feat: 타이틀 애니메이션 추가

타이틀 애니메이션 추가, TitleManager 스크립트 수정
- 화면 크기 1920x1080으로 수정
- 로딩 진행 바, 퍼센트 텍스트 비활성화 (삭제 예정), LOADING... 텍스트로 대체함
- 로딩 끝난 후 키 입력 받기 전 Title_BlinkPressAnyKeyText 애니메이션 재생
- 로딩 끝난 후 아무 키나 입력 받으면 Title_Bounce 애니메이션 재생 후 다음 씬으로 넘어감
- 전체 스크립트 파일 인코딩 형식 UTF-8로 변경, 한글 깨짐 해결
- 2d sprite 패키지 설치

ubninin committed:
Fix: 로딩 완료 텍스트 표시 문제 수정

로딩 완료 텍스트가 로딩이 완료되지 않아도 표시되는 문제 해결
- 시작 시 비활성화, 로딩 완료 후 활성화되도록 수정

Pyorang commented:
### 작성한 문제와 관련하여
1. 난 정상적으로 잘 보이는데, 그냥 오류 아닐까 생각중.
2. 일단 남겨두고 나중에 한번에 삭제하는 걸로

### 작업 내용물과 관련하여
1. 기획한 부분이랑 만들어진 부분하고 차이가 있는 것 같음 => 처음 기획 했던 내용은 로딩되면서 글씨가 왼쪽으로 압축되었다가 로딩 완료 되고 탭하면 글씨는 원상복구 되면서 캐릭터만 날라가는 걸로 생각했는데 이게 아니였남..
2. DataTableManager 삭제하기로 했는데, 일단 나중에 삭제할게요.

### 코드와 관련하여
1. 현재 코드에 주석이 너무 과도하게 많음.

예로들어,

[SerializeField] private GameObject _title;                // 타이틀 UI 오브젝트

=> 이 코드 같은 경우에도 남들이 못 알아볼까 적어놓은 것 같은데, 충분히 GameObject와 _title이라는 내용으로도 뭔지 대충 알 수 있음
=> 주석이 필요할까 싶네용?? 잘못됬다는 건 아닌데, 글의 양이 많을수록 사람이 처리해야 하는 양이 많기 때문에 빼는 걸 추천합니다

YOUKK reviewed (commented):
우리 프로젝트 코딩 컨벤션 보면 지역 변수는 카멜 케이스로 적기로 했습니다!
이거 맞춰서 써주면 좋을 거 같아요

[코딩 컨벤션 문서]
https://docs.google.com/document/d/1-BZcwiIfIA84yVELwBV30ijo36wXyAEhHrNFv4ZmKE4/edit?tab=t.kv4c42g6lis5

ubninin committed:
Design/Style: Update animation, apply naming convention in TitleManager

애니메이션 수정, TitleManager 코드 스타일 개선
- 애니메이션 움직임, 키프레임, 애니메이터 컨트롤러 구조 일부 수정
- 코드 내 변수 및 함수명을 명명 규칙에 따라 수정
